docs/development-setup.md

```diff
@@ -112,7 +112,7 @@ git checkout develop
 1. Install the required gems by running `bundle install` in the project directory.
 1. Install the required Node modules by running `yarn install` in the project directory.
 1. Run `cp config/database.yml.example config/database.yml`.
-1. Run `cp config/settings.local.yml.example config/settings.local.yml` and adjust settings as appropriate.
+1. Run `cp .env.example .env` and adjust settings as appropriate.
```

Comment:
What if we have a `.env.development` file that is checked into the repo with settings that work for development? Then we could eliminate this step. And in production, one sets up `.env` and the `.env.development` is ignored?

If development things need to be overridden, folks can use `.env.local` or `.env.development.local`.

`.env.*.local` and `.env.local` need to be added to .gitignore.